plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -535,22 +547,21 @@ struct Database {
     }
 
     /** Set mutantStatus by using MutantId.
-	Can be used both for ignoring and "accepting" mutants.
      */
-    void markMutant(ulong id, Mutation.Status to_status, string rationale = "") @trusted {
-        Mutation.Status from_status = Mutation.Status(getMutationStatus(MutationId(id)));
+    void markMutant(MutationId id, Mutation.Status to_status, string rationale) @trusted {
+        auto st_id = getMutationStatusId(id);
+        auto p = getPath(id);
 
-        // mark mutantStatus in mutationStatus-table
-        const s = format!"UPDATE %s SET status=%s WHERE id IN (SELECT st_id FROM %s WHERE id = %s)"(
-                mutationStatusTable, to_status.to!long, mutationTable, id.to!long);
-        auto stmt = db.prepare(s);
-        stmt.execute;
+        if (p.isNull)
```

Comment:
I recommend considering how to handle this failing case.
How will the user know that an invalid mutation id was specified?